client/jetpack-connect/authorize-form.jsx

```diff
@@ -10,40 +10,28 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import Main from 'components/main';
-import LoggedOutFormLinks from 'components/logged-out-form/links';
-import { getAuthorizationRemoteQueryData } from 'state/jetpack-connect/selectors';
-import { getCurrentUserId } from 'state/current-user/selectors';
-import { recordTracksEvent, setTracksAnonymousUserId } from 'state/analytics/actions';
-import EmptyContent from 'components/empty-content';
-import MainWrapper from './main-wrapper';
-import HelpButton from './help-button';
-import JetpackConnectHappychatButton from './happychat-button';
 import LoggedInForm from './auth-logged-in-form';
 import LoggedOutForm from './auth-logged-out-form';
+import MainWrapper from './main-wrapper';
+import { authQueryPropTypes } from './utils';
+import { getCurrentUserId } from 'state/current-user/selectors';
+import { recordTracksEvent, setTracksAnonymousUserId } from 'state/analytics/actions';
 
 class JetpackConnectAuthorizeForm extends Component {
 	static propTypes = {
-		authorizationRemoteQueryData: PropTypes.shape( {
-			_ui: PropTypes.string,
-			_ut: PropTypes.string,
-			client_id: PropTypes.string,
-			from: PropTypes.string,
-		} ).isRequired,
+		authQuery: authQueryPropTypes.isRequired,
+
+		// Connected props
 		isLoggedIn: PropTypes.bool.isRequired,
 		recordTracksEvent: PropTypes.func.isRequired,
 		setTracksAnonymousUserId: PropTypes.func.isRequired,
 	};
 
 	componentWillMount() {
 		// set anonymous ID for cross-system analytics
-		const { authorizationRemoteQueryData } = this.props;
-		if (
-			authorizationRemoteQueryData &&
-			authorizationRemoteQueryData._ui &&
-			'anon' === authorizationRemoteQueryData._ut
-		) {
-			this.props.setTracksAnonymousUserId( authorizationRemoteQueryData._ui );
+		const { authTracksUi, authTracksUt } = this.props;
+		if ( 'anon' === authTracksUt && authTracksUi ) {
```

Comment:
I appreciate the better naming here 👍

Reply:
😬 I actually missed this in fa2b3e74f9ff4c3a4c95141d8c9254b8fc5a3273, need to fix it up to pull from `authQuery`

Reply:
✅ 8271211a8a380bd874684ff85c5c4b344f5f2d75